index.html

```diff
@@ -0,0 +1,455 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+    <script type="text/javaScript" src="js/main.js"></script>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" onclick="showNav()">
+        <i class="fas fa-bars"></i>
+      </button>
```

Comment:
- [ ] I think it could be better to add the menu button in the `nav` tag, this is because the menu button represents the links in the mobile version